selfupdate.go

```diff
@@ -3,48 +3,44 @@ package main
 import (
 	"bufio"
 	"fmt"
+	"log"
+	"os"
+
 	"github.com/blang/semver"
 	"github.com/lawrencegripper/azbrowse/version"
 	"github.com/rhysd/go-github-selfupdate/selfupdate"
-	"log"
-	"os"
 )
 
 func confirmAndSelfUpdate() {
 	latest, found, err := selfupdate.DetectLatest("lawrencegripper/azbrowse")
 	if err != nil {
-		log.Println("Error occurred while detecting version:", err)
-		return
+		log.Panicln("Error occurred while detecting version:", err)
```

Comment:
I think this would mean the util wouldn't be usable if github was down which feels bad. One option is that we could use a 'ReadString' after the original 'Println' for the user to acknowledge the failure and ensure the log lines aren't lost. 

What do you think?

Reply:
Ah, good point!